app/helpers/layout_helper.rb

```diff
@@ -322,4 +322,12 @@ def number_f(f, attr, options = {})
       f.number_field attr, options
     end
   end
+
+  private
+
+  def authorized_associations(associations)
+    associations.authorized
+  rescue (NoMethodError)
```

Comment:
I ran into this in the Filter edit page (but it might be on pther pages as well).
For some strange reason `associations.responds_to?(:authorized)` (which I tried at first as guard) returned true but it failed with NoMethodError nonetheless.